4-domain.go

```diff
@@ -0,0 +1,109 @@
+package domain
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop/runtime"
+	"github.com/nspcc-dev/neo-go/pkg/interop/storage"
+)
+
+// Main is a very useful function.
+func Main(operation string, args []interface{}) interface{} {
+	ctx := storage.GetContext()
+
+	// Queries the domain owner
+	if operation == "query" {
+		if checkArgs(args, 1) {
```

Comment:
I think
```
if len(args) != 1 {
    return false
}
```
would be a little more goesque. And it will allow to remove `checkArgs()`.